ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -8,113 +8,181 @@
       regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCC', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CCC', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CCC', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CCC', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
       run: function(data) {
         data.phase = 'phase2';
       },
     },
     {
       id: 'VarisEx Clones',
       regex: Regexes.ability({ source: 'Phantom Varis', id: '4CB3', capture: false }),
+      regexDe: Regexes.ability({ source: 'Varis-Doppelgänger', id: '4CB3', capture: false }),
+      regexFr: Regexes.ability({ source: 'double de Varis', id: '4CB3', capture: false }),
+      regexJa: Regexes.ability({ source: 'ヴァリスの幻影', id: '4CB3', capture: false }),
       run: function(data) {
         data.clones = 'active';
       },
     },
     {
       id: 'VarisEx Ignis Est',
       regex: Regexes.startsUsing({ source: 'Ignis Est', id: '4CB6', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CB6', source: 'Ignis Est', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CB6', source: 'Ignis Est', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CB6', source: 'イグニス・エスト', capture: false }),
       delaySeconds: 2,
       response: Responses.getOut(),
     },
     {
       id: 'VarisEx Ventus Est',
       regex: Regexes.startsUsing({ source: 'Ventus Est', id: '4CC7', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CC7', source: 'Ventus Est', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CC7', source: 'Ventus Est', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CC7', source: 'ウェントゥス・エスト', capture: false }),
       delaySeconds: 2,
       response: Responses.getIn('info'),
     },
     {
       id: 'VarisEx Altius',
       regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCA', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CCA', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CCA', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CCA', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
       infoText: {
         en: 'Bait Slashes',
+        ja: '縦へ、アルティウスを誘導',
+        cn: 'Boss身后诱导剑气方向',
       },
     },
     {
       id: 'VarisEx Citius',
-      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CF0' }),
+      regex: Regexes.startsUsing({ id: '4CF0', source: 'Varis Yae Galvus' }),
+      regexDe: Regexes.startsUsing({ id: '4CF0', source: 'Varis yae Galvus' }),
+      regexFr: Regexes.startsUsing({ id: '4CF0', source: 'Varis yae Galvus' }),
+      regexJa: Regexes.startsUsing({ id: '4CF0', source: 'ヴァリス・イェー・ガルヴァス' }),
       alertText: function(data, matches) {
-        if (data.me == matches.target) {
+        const target = matches.target;
```

Comment:
Are you should this should be `const`? I would have expected it to be written as `let target = matches.target;`.

Also, what sort of undefined error were you getting? The log lines the `startsUsing` regex matches against always include a target.

Reply:
Probably more things could/should be `const` instead of `let` in cactbot, if I'm being honest.  I think this is fine, as-is?

Reply:
That's fair. I am a little concerned that this was necessary though. I'm not sure what would be undefined to cause a bug here. There is an oddity in that the log shows the `14 startsCasting` line, then a `17 cancelCast` line before two `15` lines, one for each tank. I should hunt through and see what happens if there's only one target hit.

Reply:
Thas's because I copied those localized strings from the other file, and in this file, it doesn't declare a `target` variable, So I have to declare one above, so I was not necessary to change the lines below. 
As the `const`, I thought the `target` variable would not (or should not) be changed, so I use `const` instead of `let`.

Reply:
Ahh, right, I totally skipped over the fact that this copy of the localizations is *not* `matches.target`. My fault, I should have read more carefully!